### ISSUE TITLE
[feaLib] allow compiling features to existing layout tables?

### ISSUE THREAD

**Issue by khaledhosny** [behdad/fonttools#599]
In FontForge when compiling a feature file to a font, it does not overwrite existing layout tables but amends the new features to them (unlike makeotf and feaLib). This is really very convenient to e.g. add a new feature to an existing fonts without having to dump all existing layout tables to feature syntax then apply them again. I use it also to have separate files for different scripts, so that I can have `languagesystem` statements without registering all features in the font to all language systems (the alternative of explicitly adding `script` and `language` statements to each feature is cumbersome to write).

I don’t know yet how hard is it to extend feaLib to do this, but if this kind of feature is acceptable I’ll give it a try.

**Comment by behdad** [comment]
Yes, we should allow that.  We'll take a look next week in Berlin.

**Comment by justvanrossum** [comment]
I'm currently running into this, too: I'm using feaLib to compile a complex `kern` feature, but (for now) I'd prefer the rest of the features to be generated by the FDK. Cc. @cjdunn

**Comment by justvanrossum** [comment]
Btw. `addOpenTypeFeatures()` and `addOpenTypeFeaturesFromString()` have a `tables` argument these days, that solves this issue at least partially. See https://github.com/fonttools/fonttools/commit/b2da85cffd918195c45a973bf28cdd4f2f77673e

**Comment by yisibl** [comment]
Yes, this is very useful for adding `chws / vchw`: https://github.com/kojiishi/east_asian_spacing/issues/175#issuecomment-1777089788
